fix: Remove the loop condition in the test code to fix CI

ShingmoYeung opened the pull request:
Remove the loop condition in the test code.

Fix: https://github.com/jcasbin/lettuce-redis-watcher/issues/2

ShingmoYeung committed:
[NEW] Added maven-ci.yml
[NEW] Added WatcherConstant.java
[NEW] Added LettuceRedisWatcher.java
[NEW] Added LettuceSubscriber.java
[NEW] Added LettuceSubThread.java
[NEW] Added LettuceRedisWatcherTest.java
[UPDATE] Updated .gitignore
[NEW] Added .releaserc.json
[NEW] Added maven-settings.xml
[NEW] Added pom.xml
[UPDATE] Updated README.md

ShingmoYeung committed:
Refactor LettuceRedisWatcherTest.java to use local IP address instead of remote IP address for testing.

ShingmoYeung committed:
Update LettuceRedisWatcherTest.java to simplify watcher initialization and update logic.

hsluoyz commented:
@ShingmoYeung plz fix conflicts

![image](https://github.com/jcasbin/lettuce-redis-watcher/assets/3787410/6662db6c-f361-4e54-8677-548e30360d8d)

ShingmoYeung commented:
> @ShingmoYeung plz fix conflicts
> 
> ![image](https://user-images.githubusercontent.com/3787410/258956599-6662db6c-f361-4e54-8677-548e30360d8d.png)

@hsluoyz permission denied?

![image](https://github.com/jcasbin/lettuce-redis-watcher/assets/14231553/297d717d-3f18-485c-8295-1bcfaab0e306)
![image](https://github.com/jcasbin/lettuce-redis-watcher/assets/14231553/a39590ca-6bb4-4dd9-87d1-3b7979792437)

hsluoyz commented:
@ShingmoYeung it's your PR. You of course have rights to modify it. If you cannot do it by web, you can modify your commits in git

ShingmoYeung committed:
Merge branch 'master' into feature-add-lettuce

# Conflicts:
#	src/test/java/org/casbin/test/LettuceRedisWatcherTest.java

ShingmoYeung committed:
Refactor Redis connection initialization and add support for Redis Cluster.

- Add Redis URI constants.
- Update Redis connection initialization in LettuceRedisWatcher.
- Add constructors for different Redis connection types.
- Add support for Redis Cluster in LettuceRedisWatcher.
- Update LettuceRedisWatcherTest to test Redis Cluster connection.

ShingmoYeung committed:
Update LettuceRedisWatcherTest.java to use localhost as the redis server instead of the specific IP address.

ShingmoYeung commented:
> @ShingmoYeung it's your PR. You of course have rights to modify it. If you cannot do it by web, you can modify your commits in git

@hsluoyz fixed it, code-review please

hsluoyz commented:
@ShingmoYeung plz fix:

![image](https://github.com/jcasbin/lettuce-redis-watcher/assets/3787410/ebd39402-259c-45a1-9d7f-7d6a95900213)

ShingmoYeung committed:
Update node version from 16 to 18 in maven-ci.yml and add new semantic.yml.

ShingmoYeung committed:
Refactor semantic.yml to disable validation for the PR title and all the commits.
